lib/rack/tracker/metrigo/metrigo.rb

```diff
@@ -0,0 +1,35 @@
+class Rack::Tracker::Metrigo < Rack::Tracker::Handler
+  class Event < OpenStruct
+    def initialize(args)
+      args[:arguments] ||= {}
+      super
+    end
+
+    def write
+      "DELIVERY.DataLogger.#{function_name}(#{write_arguments})"
+    end
+
+    def write_arguments
+      arguments.to_json  if arguments.present?
+    end
+  end
+
+  self.position = :body
+
+  def render
+    Tilt.new( File.join( File.dirname(__FILE__), 'template/metrigo.erb') ).render(self)
+  end
+
+  # merges shop_id into all Event objects
+  def events
+    super.map do |event|
+      event.arguments.merge!(shop_id: options[:shop_id])
+      event
+    end
+  end
+
+  def self.track(name, *event)
+    { name.to_s => [{ class_name: 'Event', function_name: event[0].to_s.camelize(:lower), arguments: event[1] }] }
```

Comment:
i think i can't, since this is a class method and the options are on the handler instance.  or am I missing something?